src/v1_0/rw/elog.cpp

```diff
@@ -132,11 +146,14 @@ namespace abb ::rws ::v1_0 ::rw ::elog
         {
             std::stringstream arg_name;
             arg_name << "arg" << i;
-            auto arg_node = xmlFindNodes(node, XMLAttribute("class", arg_name.str()))[0];
-
-            std::string value = arg_node->innerText();
-            std::string valueType = xmlNodeGetAttributeValue(arg_node, "type");
-
+            auto arg_nodes = xmlFindNodes(node, XMLAttribute("class", arg_name.str()));
+            std::string value = "STRING";
```

Comment:
Here maybe let's just keep an empty string to comply with how other parsers behave

Reply:
The `ElogMessageArg` is an alias for `std::variant<std::string, long, float>` so it must be one one: `"STRING"`, `"LONG"` or `"FLOAT"`. 
If we want to have either the "UNDEFINED" or empty string value, the data type must be a string.

Reply:
My mistake, it should be the other way. The `ElogMessageArg` is an alias for `std::variant<std::string, long, float>` so  the `value_type` it must be one one: `"STRING"`, `"LONG"` or `"FLOAT"`.
I think leaving `"UNDEFINED"` would make it easier to spot in the log message that something is missing, but on other hand it could be the expected behaviour it's left blank...